### PR TITLE
html-vas-response: add option for discrete scale points

### PR DESCRIPTION
**Pull request by kinleyid**
Hello, I've updated the package at the request of a user. Please let me know if I've done anything incorrectly with this pull request!

### CODE REVIEW

**Comment by jodeleeuw** [comment]
Hi @kinleyid,

Thanks for this contribution!

Instead of manually bumping the version number, please add a changeset following the instructions from the bot above. This will then auto bump the version when we merge and release.

**Comment by kinleyid** [comment]
I believe I've just added a changeset---please let me know if anything is missing!

File: packages/plugin-html-vas-response/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@jspsych-contrib/plugin-html-vas-response",
-  "version": "1.1.0",
+  "version": "1.2.0",
```
Comment (jodeleeuw) [inline packages/plugin-html-vas-response/package.json:3]: Can you revert this change, so that the changeset action will autobump to 1.2?

File: .changeset/eighty-dolphins-approve.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"@jspsych-contrib/plugin-html-vas-response": patch
```
Comment (jodeleeuw) [inline .changeset/eighty-dolphins-approve.md:2]: This should be a `minor` version instead of a `patch`, since you are adding a new feature

File: .changeset/eighty-dolphins-approve.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+---
+
+html-vas-response: add option for discrete scale points
```
Comment (bjoluc) [inline .changeset/eighty-dolphins-approve.md:5]:
```suggestion
Add option for discrete scale points
```
No need to mention `html-vas-response` here since what you write here only appears in the changelog and release notes of `html-vas-response` :slightly_smiling_face:

**Comment by kinleyid** [comment]
Just finished making the requested changes!

**Comment by jodeleeuw** [comment]
Thanks! And thank you for the contribution!
